pages/mesosphere/dcos/1.14/installing/production/upgrading/index.md

```diff
@@ -264,9 +264,11 @@ Proceed with upgrading every master node one at a time in any order using the fo
         [enterprise type="inline" size="small" /]
 
     1.  Verify that `/opt/mesosphere/bin/mesos-master --version` indicates that the upgraded master is running the version of Mesos specified in the [release notes](/mesosphere/dcos/1.14/release-notes/), for example `1.5.1`.
-	1.  Verify that the number of under-replicated ranges has dropped to zero as the IAM database is replicated to the new master. This can be done by running the following command and confirming that the last column on the right shows only zeros. [enterprise type="inline" size="small" /]
+
+1.  Verify that the number of under-replicated ranges has dropped to zero as the IAM database is replicated to the new master. This can be done by running the following command and confirming that the last column on the right shows only zeros. [enterprise type="inline" size="small" /]
```

Comment:
Does this need to be indented the same as the previous line?

Reply:
The wording of "Verify that /opt/mesosphere/bin/mesos-master --version..." appears to refer to an upgraded master. Since these instructions are about validating an upgrade, it seems reasonable to subsume them under the top level step 3, "Validate the upgrade". Thus there are four steps to validate an upgrade. 

How would you number them differently?

Reply:
@jongiddy we are still waiting on merging this info. Is there a way you would number them differently?

Reply:
No, I wasn't sure why the indentation had changed, but I'm happy with the explanation.